src/ripple/app/tx/impl/InvariantCheck.h

```diff
@@ -342,6 +353,19 @@ class ValidNFTokenPage
         beast::Journal const&);
 };
 
+/**
+ * @brief Invariant: Validates counts of NFTokens after all transaction types.
+ *
+ * The following checks are made:
+ *  - The number of minted or burned tokens may only be changed by
```

Comment:
For clarity, I avoid the use of _may_, using _can_ or _might_ instead. I would say "...burned tokens can only be changed...."